doc/sphinx/compiling.rst

```diff
@@ -32,69 +33,267 @@ Installation Prerequisites
+  optimized for your hardware. The source code for SUNDIALS will be downloaded
+  and compiled with the rest of Cantera, so installing from your distribution's
+  repositories is optional.
+
+* The minimum compatible Cython version is 0.23. If your distribution does not
+  contain a suitable version, you may be able to install a more recent version
+  using ``pip``.
+
+* Building the Fortran interface also requires ``gfortran`` or another supported
+  Fortran compiler.
+
+* Users of other distributions should install the equivalent packages, which
+  may have slightly different names.
+
+.. _sec-ubuntu-debian-reqs:
+
+Ubuntu & Debian
+^^^^^^^^^^^^^^^
+
+* Ubuntu 12.04 LTS (Precise Pangolin) or newer is required; 16.04 LTS (Xenial Xerus)
+  or newer is recommended
+
+* Debian 7.0 (Wheezy) or newer; 8.0 (Jessie) or newer is recommended
+
+* The following packages must be installed to build any of the Cantera modules using
   your choice of package manager::
 
-      g++ python scons libsundials-serial-dev libboost-dev
+      g++ python scons libboost-dev
 
-* Building the python module also requires::
+* In addition to the general packages, building the Python 2 module also requires::
 
       cython python-dev python-numpy python-numpy-dev python-setuptools
 
-* Building the python3 module requires the following libraries::
+* In addition to the general packages, building the Python 3 module also requires::
+
+      cython python3 python3-dev python3-setuptools python3-numpy
 
-      cython python3-dev python3-setuptools python3-numpy
+.. _sec-fedora-reqs:
 
-* For Fedora (version 22 or higher) users, the following packages should
-  be installed via the package manager::
+Fedora & RHEL
+^^^^^^^^^^^^^
 
-      gcc-c++ python scons sundials-devel blas-devel lapack-devel boost-devel
+* The following packages must be installed to build any of the Cantera modules using
+  your choice of package manager::
 
-  If your Fedora version is lower than 22, the `sundials-devel` package is not
-  available, and you should build Sundials from source.
+      gcc-c++ python scons boost-devel
 
-  Python module required packages are::
+* In addition to the general packages, building the Python 2 module also requires::
 
       python-setuptools python-devel Cython numpy
 
-* Checking out the source code from version control requires Git (install
-  ``git``).
+* In addition to the general packages, building the Python 3 module also requires::
 
-* The minimum compatible Cython version is 0.23. If your distribution does not
-  contain a suitable version, you may be able to install a more recent version
-  using `pip`.
+      python3 python3-setuptools python3-devel Cython python3-numpy
 
-* Building the Fortran interface also requires `gfortran` or another supported
-  Fortran compiler.
-* Users of other distributions should install the equivalent packages, which
-  may have slightly different names.
+.. _sec-opensuse-reqs:
+
+OpenSUSE & SEL
```

Comment:
This should be "SLE", not "SEL", right? Maybe it would be better to write "SUSE Linux Enterprise" as I had no idea what this is and SUSE is not the top hit for either of those acronyms (unlike RHEL in the previous case).